src/apps/loyalty/server/index.tsx

```diff
@@ -34,7 +34,7 @@ app.use(session({
   cookie: {},
 }))
 app.use(artsyPassport(Object.assign({}, process.env, {
-  CurrentUser,
+  CurrentUser: Backbone.Model,
```

Comment:
We just need this so `get` and `set` 'work' in the passport lib...I wonder if that can be relaxed? https://github.com/artsy/artsy-passport/issues/104